RHDEVDOCS-2831: Improvements for deployment docs (#61662)
openshift/openshift-docs: applications/deployments/deployment-strategies.adoc

toc::[]

> _Deployment strategies_ are used to change or upgrade applications without downtime, in a way that users barely notice a change.

Reviewer: ```suggestion
_Deployment strategies_ are used to change or upgrade applications without downtime so users barely notice a change.
```
- Maybe this edit would make the sentence a bit more streamlined? Your call whether to make the change.
